src/styles/Extn-NavigationAndHistory.less

```diff
@@ -236,4 +236,31 @@
 #mrof-container .footer {
   background-color: #2c2c2c;
   border-top: 1px solid #343434;
+}
+
+#mrof-list li {
+  position: relative;
+}
+
+.remove-file {
+  color: #adb9bd;
+  position: absolute;
+  right: 8px;
+  top: 50%;
+  transform: translateY(-50%);
+  cursor: pointer;
+  opacity: 0.6;
+  font-size: 20px;
+  font-weight: 500;
+  display: none;
+}
+
+#mrof-list li:hover .remove-file {
+  color: #fff;
```

Comment:
add the hover close button before the list so that its eazier to visually associate. 
![image](https://github.com/user-attachments/assets/759368f0-a589-4226-89b7-a8e6ba27df64)